src/dataprep/main/prep_nsf/scinet_data_to_db.py

```diff
@@ -55,12 +55,14 @@
 def load_scinet(filepath):
     df = pd.read_csv(filepath, 
                         sep="\t",
-                        names=["NSF_Award_Number", "PaperID", "Type", "Diff_ZScore"])
+                        names=["NSF_Award_Number", "PaperID", "Type", "Diff_ZScore"], 
+                        skiprows=1)
     df.drop_duplicates(inplace=True)
 
     # Create the GrantID column by removing non-numeric characters and formatting
     
     df['GrantID'] = df['NSF_Award_Number'].str.extract(r'-(\d+)') 
```

Comment:
Why is here a new variable created? Why not just load the table as it is? 

Also why drop the diff_zscore and type?  These are important to figure out confidence of the link

Reply:
new vairable: I mean GrantId, why not just keep the original NSF_Award_Number